evaluator/src/main/kotlin/PackageRule.kt

```diff
@@ -162,6 +162,18 @@ open class PackageRule(
 
         override fun issueSource() = "$name - ${pkg.id.toCoordinates()} - $license ($licenseSource)"
 
+        /**
+         * A [RuleMatcher] that checks if a [detected][LicenseSource.DETECTED] license is excluded. This is the case if
+         * all keys of [licenseFindings] are associated to at least one [PathExclude].
+         */
+        fun isExcluded() =
+            object : RuleMatcher {
+                override val description = "isExcluded($license)"
+
+                override fun matches() =
+                    licenseSource != LicenseSource.DETECTED || licenseFindings.values.all { it.isNotEmpty() }
```

Comment:
I believe the current implementation makes `isExcluded()` always match for declared or concluded licenses. Shouldn't is be the opposite, i.e. declared or concluded licenses should never be excluded (by path excludes)?

Reply:
Thanks, fixed.